courses/machine_learning/deepdive2/structured/solutions/3b_bqml_linear_transform_babyweight.ipynb

```diff
@@ -0,0 +1,1162 @@
+{
```

Comment:
<p>* Prefix this cell with "Let's compute the rmse from the evaluation table we saved into BiqQuery:"</p><p>* Instead of recomputing the evaluation table, reused the one saved in the previous cell with the --destination_table arguement</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/669/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Done</p>